R/theme_element.R

```diff
@@ -20,28 +20,41 @@ element_style <- function(...){
 
 #' Element Align
 #'
-#' @param left Variables to align to the left
-#' @param right Variables to align to the right
-#' @param char Variable to align on a provided character
-#' @param char_val Vector of one or more characters to align on. If NULL, data values in `char` variable(s) will be aligned on the first occurrence of a decimal place or space. If more than one
-#' character is provided, alignment will be based on the first occurrence of any of the characters. For alignment based on white space, leading white spaces will be ignored.
+#' @param align Alignment to be applied to column. Acceptable values: "left" for left alignment, "right" for right alignment",
```

Comment:
This is kind of dumb but just FYI if you use shift+ctrl+/ it will auto enter your comments so the test doesn't go beyond 80 characters.